python/python/tests/test_pydantic.py

```diff
@@ -412,3 +412,23 @@ class TestModel(LanceModel):
 
     t = TestModel(id=1)
     assert t.vectors == [[0.0] * 16]
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 11),
+    reason="StrEnum where only introduced in python 3.11",
+)
+def test_str_enum():
+    from enum import StrEnum
+
+    class TestModel(pydantic.BaseModel):
+        a: StrEnum
```

Comment:
Is this how it's used? I would think it would be something like:

```suggestion
    class Status(StrEnum):
        Pending = "pending"
        Running = "running"
        Finished = "finished"

    class TestModel(pydantic.BaseModel):
        a: Status
```

Reply:
yes, much better!